drivers/SmartThings/zwave-switch/profiles/ge-dimmer-switch.yml

```diff
@@ -0,0 +1,24 @@
+name: ge-dimmer-switch
+components:
+  - id: main
+    capabilities:
+      - id: switch
+        version: 1
+      - id: switchLevel
+        version: 1
+      - id: refresh
+        version: 1
+    categories:
+      - name: Switch
+preferences:
+  - name: "ledIndicator"
+    title: "LED Indicator"
+    description: "Turn LED indicator on..."
+    required: false
+    preferenceType: enumeration
+    definition:
+      options:
+        0: "off"
+        1: "on"
+        2: "never"
```

Comment:
the text for the preference should match the text from the DTH, "on", "off", "never" are the keys for the enum, but the display values are "When Off", "When On", and "Never"

Reply:
> the text for the preference should match the text from the DTH, "on", "off", "never" are the keys for the enum, but the display values are "When Off", "When On", and "Never"

I think that here is the place where the keys for the enum set:
`void indicatorWhenOn() {
	sendEvent(name: "indicatorStatus", value: "when on", displayed: false)
	sendHubCommand(new physicalgraph.device.HubAction(zwave.configurationV1.configurationSet(configurationValue: [1], parameterNumber: 3, size: 1).format()))
}

void indicatorWhenOff() {
	sendEvent(name: "indicatorStatus", value: "when off", displayed: false)
	sendHubCommand(new physicalgraph.device.HubAction(zwave.configurationV1.configurationSet(configurationValue: [0], parameterNumber: 3, size: 1).format()))
}

void indicatorNever() {
	sendEvent(name: "indicatorStatus", value: "never", displayed: false)
	sendHubCommand(new physicalgraph.device.HubAction(zwave.configurationV1.configurationSet(configurationValue: [2], parameterNumber: 3, size: 1).format()))
}`

Reply:
It's actually here: https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/3cc8efa81eee0ac4c2a9ed420e1a6c25f88dedf0/devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy#L49

that line is for a custom attribute

Reply:
you can also see the keys (as opposed to the values) for enum being used here: https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/3cc8efa81eee0ac4c2a9ed420e1a6c25f88dedf0/devicetypes/smartthings/dimmer-switch.src/dimmer-switch.groovy#L92

off/on/never wouldn't display with the actual device